data/json/items/book/dodge.json

```
@@ -44,6 +113,28 @@
     "category": "manuals",
     "name": { "str": "Break a Leg!", "str_pl": "copies of Break a Leg!" },
     "description": "The Kids' Guide to Acting and Stagecraft.",
+    "variants": [
+      {
+        "id": "manual_dodge_kid",
+        "name": { "str": "Break a Leg!", "str_pl": "copies of Break a Leg!" },
+        "description": "The Kids' Guide to Acting and Stagecraft.",
+        "weight": 1
+      },
+      {
+        "id": "manual_dodge_kid_1",
+        "name": { "str": "Self-defense for Your Child", "str_pl": "copies of Self-defense for Your Child" },
+        "description": "Fully illustrated practical course of simple techniques to defend your child from enemy.  You doubt it will help much for real child, but you found some interesting info here.",
+        "//": "978-0874070248",
+        "fun": 0,
+        "weight": 1
+      },
+      {
+        "id": "manual_dodge_kid_2",
+        "name": { "str": "Protective Measures: A Graphic Novel", "str_pl": "copies of Protective Measures: A Graphic Novel" },
+        "description": "Modern comics about unnamed Hero, that protect people without any superpowers, using only his body and different marital arts.  The draw is good, and shows the movement and strategy of each character, but the plot is fairly simple.",
```

```suggestion
        "name": { "str": "Protective Measures: A Graphic Novel", "str_pl": "copies of Protective Measures: A Graphic Novel" },
        "description": "Modern comics about unnamed Hero, that protects people without any superpowers, using only his body and different martial arts.  The artwork is well-illustrated and shows the movement and strategy of each character, but the plot is fairly simple.",
```
Is this a fake book, since no ISBN?

~~Also I don't like the implications of mar*ital* arts 😛~~

---

Yeah, i thought to add "five ring:graphic novel", but with this there will be 3 different five rings books - as MA, as cut, and as dodge. I also was not able to find something good to replace it, so i made it instead